src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/VmrCommandLineOptions.cs

```diff
@@ -24,17 +24,26 @@ internal abstract class VmrCommandLineOptions : CommandLineOptions
     public IServiceCollection RegisterServices()
     {
         var tmpPath = Path.GetFullPath(TmpPath ?? Path.GetTempPath());
-        var localDarcSettings = LocalSettings.LoadSettingsFile();
+        string gitHubToken = null;
+        string azureDevOpsToken = null;
+
+        try
+        {
+            var localDarcSettings = LocalSettings.LoadSettingsFile(this);
+            gitHubToken = GitHubPat ?? (localDarcSettings?.GitHubToken);
+            azureDevOpsToken = AzureDevOpsPat ?? (localDarcSettings?.AzureDevOpsToken);
+        }
+        catch (DarcException)
+        {
+
```

Comment:
Please put a comment inside of this catch block saying something like we want to allow null